XIVSlothCombo/Combos/SGE.cs

```diff
@@ -195,7 +195,7 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
     //Can replace Zero Charges/Stacks of Phlegma with Toxikon (if you can use it) or Dyskrasia 
     internal class SagePhlegmaFeature : CustomCombo
     {
-        protected internal override CustomComboPreset Preset { get; } = CustomComboPreset.SagePhlegmaToxikonFeature;
+        protected internal override CustomComboPreset Preset { get; } = CustomComboPreset.SagePhlegmaFeature;
```

Comment:
Have you accounted for this Feature name-change in `CustomComboPreset.cs`?
I don't see any changes to the file in this PR currently.